.github/workflows/jib-cli-release.yml

```diff
@@ -0,0 +1,56 @@
+name: Release Jib CLI
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: new release version
+        required: true
+        default: (for example, 0.1.0)
+
+jobs:
+  release:
+    name: Release Jib CLI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Build Jib CLI
+        run: |
+          # TODO: run integration test? (Requries auth with GCP.)
+          ./gradlew clean build jib-cli:instDist
+
+      - name: Run Gradle release
+        run: |
+          git checkout -b cli_release_v${{ github.event.inputs.release_version }}
+          git config user.email "chanseok@google.com"
```

Comment:
I wonder if this can be configured to be the user triggering the event?

Reply:
Yeah, I need to figure this out eventually.